diff --git a/node_modules/nuxt-feedme/dist/module.mjs b/node_modules/nuxt-feedme/dist/module.mjs
index 3547f54..3b73b75 100644
--- a/node_modules/nuxt-feedme/dist/module.mjs
+++ b/node_modules/nuxt-feedme/dist/module.mjs
@@ -78,9 +78,7 @@ const pkg = {
 };
 
 const defaultFeedmeContentOptions = {
-  collections: {
-    content: (x) => x.all()
-  },
+  collections: {},
   mapping: void 0,
   // METADATA is now contains in all: root, root.seo and root.meta
   // According to algorithm, priority makes difference. So all in feedme must be first to allow override default values
